src/electronApp/angularApp/services/printerService.ts

```diff
@@ -110,7 +118,9 @@ export class PrinterService implements IPrinterService {
     /** @inheritdoc */
     public GetPrinterStatusAsync(): Promise<PrinterStatus> {
         if (this.printer == null) {
-            throw new Error('Cannot call this method before calling and awaiting ConnectAsnc()');
+            throw new Error(
+              "Cannot call this method before calling and awaiting ConnectAsync()"
+            );
```

Comment:
Any reason for the extra new lines with these?

Reply:
That's from the formatter I'm using.

Reply:
Most of my liniting was done by TSLint, for things like curly brace spacing. I don't have any specific formatter for new lines on method calls like this.

I'm not way precious about the exact whitespace formatting, if we want to pick a system and stick to it. In this case, this adds quite a bit of length to each each method in the file, so I don't think the extra spaces are really adding anything. If you're happy remove them, I'll get this merged :)

Reply:
I looked around in the settings of the formatter and the new lines are because it was trying to fit the code to a specific width.